### ISSUE TITLE
repo name, location

### ISSUE THREAD

**Issue by pchalasani** [Arvinds-ds/tf_logbrain#1]
Once you join, this would be under the MediaMath organization. And a silly detail, but the name `tf_logbrain` seems to indicate it will only be logistic models, but I see a folder `linear_models`. If we anticipate extensions, it's probably better to call it `tf_brain`, but that can be done once the repo moves to the MediaMath org.

**Comment by Arvinds-ds** [comment]
Agree, Actually I have a repo tfbrain (private) to house extensions for later..but I pushed this out first as this will be the focus to get it right. The idea of this repo is to was to showcase end to end Tensorflow minimal code from model production to serving
